integration-tests/src/integrationTest/org/apache/cassandra/sidecar/coordination/ClusterLeaseClaimTaskIntegrationTest.java

```diff
@@ -115,32 +111,35 @@ void setup()
     @AfterAll
     static void cleanup()
     {
-        TestResourceReaper.create().with(vertx).close();
+        getBlocking(vertx.close(), 10, TimeUnit.SECONDS, "Close vertx");
```

Comment:
Sad to see moving away from `TestResourceReaper` :|

Reply:
There are opportunities to consolidate common test code in the test-common subproject. Maybe something to consider for future improvements